### PR TITLE
feat: add ompp image

### PR DESCRIPTION
**Pull request by vexingly**
Very basic image for running MPI jobs provided by OpenM++ (https://github.com/openmpp/docker/blob/master/ompp-run-ubuntu/Dockerfile)

### CODE REVIEW

File: ompp-run-ubuntu/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+#     -it openmpp/openmpp-run:ubuntu \
+#     bash
+
+FROM ubuntu:22.04
+
+# disable debconf terminal input
+ARG DEBIAN_FRONTEND=noninteractive
+
+# update base image
+RUN apt-get update
```
Comment (Souheil-Yazji) [inline ompp-run-ubuntu/Dockerfile:39]: All these `apt` related RUN commands can be consolidated to 1 layer. https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments 
Will save on layers.

File: ompp-run-ubuntu/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+RUN apt-get install -y apt-utils
+RUN apt-get upgrade -y
+
+# install Open MPI and SQLite
+RUN apt-get install -y sqlite3 && \
```
Comment (Souheil-Yazji) [inline ompp-run-ubuntu/Dockerfile:44]: can possibly be done above

File: ompp-run-ubuntu/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  apt-get install -y openmpi-bin && \
+  apt-get autoclean
+
+# download and install unixODBC (optional)
+RUN apt-get install -y unixodbc && \
```
Comment (Souheil-Yazji) [inline ompp-run-ubuntu/Dockerfile:49]: can possibly be done above

File: ompp-run-ubuntu/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+# Done with installation
+# set user, work directory and entry point
+#
+ARG OMPP_USER=ompp
+ARG OMPP_GROUP=ompp
+ARG OMPP_UID=1999
+ARG OMPP_GID=1999
+
+ENV OMPP_USER  ${OMPP_USER}
```
Comment (Souheil-Yazji) [inline ompp-run-ubuntu/Dockerfile:69]: Can use multi-line and only 1 intermediate image. See example here https://docs.docker.com/engine/reference/builder/#env

**Comment by vexingly** [comment]
@Souheil-Yazji I did a bit of cleanup per your suggestions!

**Review by Souheil-Yazji — APPROVED** [review]
LGTM

**Pull request merged**